DsaPlayGround/PlayGround/MethodOverloading_AssessmentTwo/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MethodOverloading_AssessmentTwo
+{
+    class Program
+    {
+        static void Main()
+        {
+            try
+            {
+                Console.Write("Enter 3 digits: ");
+                string input = Convert.ToInt32(Console.ReadLine()).ToString();
```

Comment:
nice

Reply:
Comment addressed, thanks for pointing it out.